test/nodejs/route.test.js

```diff
@@ -0,0 +1,31 @@
+const test = require('tape');
+var Valhalla = require('../../');
+
+var config = require('./fixtures/basic_config');
+
+test('route: can get a route in Hershey', function(assert) {
+  var valhalla = new Valhalla(JSON.stringify(config));
+  var hersheyRequest = '{"locations":[{"lat":40.546115,"lon":-76.385076,"type":"break"}, {"lat":40.544232,"lon":-76.385752,"type":"break"}],"costing":"auto"}';
+  var route = JSON.parse(valhalla.route(hersheyRequest));
+  assert.ok(route);
+  assert.equal(route['trip']['status_message'], 'Found route between points', 'route was found');
+  assert.equal(route['trip']['legs'].length, 1, 'Route has one leg');
+  assert.equal(route['trip']['legs'][0]['maneuvers'].length, 2, 'Leg has two maneuvers');
+  assert.end();
+});
+
+test('route: returns an error if no edges found', function(assert) {
+  var valhalla = new Valhalla(JSON.stringify(config));
+  var hersheyRequest = '{"locations":[{"lat":5,"lon":-76.385076,"type":"break"}, {"lat":40.544232,"lon":-76.385752,"type":"break"}],"costing":"auto"}';
+  assert.throws(() => { valhalla.route(hersheyRequest) }, /No suitable edges near location/, 'Throws correct error');
+  assert.end();
+});
+
+test('route: returns an error if request format is wrong', function(assert) {
+  var valhalla = new Valhalla(JSON.stringify(config));
+  var hersheyRequest = '{"locations":[40.546115,-76.385076], [40.544232,"lon":-76.385752],"costing":"auto"}';
+  // NOTE: we are not throwing a very useful error in this case - we should track this
```

Comment:
do we have a project consistent `todo` tag? I keep seeing a lot of
```
// TODO
```
maybe we should use that here too instead of `// NOTE`